aws-api-appsync/src/main/java/com/amplifyframework/api/aws/appsync/SelectionSet.java

```diff
@@ -87,6 +91,15 @@ public SelectionSet(String value, Set<SelectionSet> nodes) {
      */
     @Override
     public String toString() {
+        return toString("");
+    }
+
+    /**
+     * Generates the String value of the SelectionSet for a GraphQL query document.
+     * @param margin a margin with which to prefix each field of the selection set.
+     * @return String value of the SelectionSet for a GraphQL query document.
+     */
+    public String toString(String margin) {
```

Comment:
> * `@Override`

I do have @Override on the `toString()` method, but this is `toString(String margin)`, so it's not overriding anything.

> Passing thought -- I wonder if we should have made an explicit `buildGraphQlDocument()` or something. `toString()` can often just have a pretty-print of the object state, which isn't necessarily in any particular standard form. Not really a big deal.

I thought about this too.  But I decided to just use `toString()` because I would still need to override `toString()` anyway to provide a useful description for debugging purposes, and `toString()` would just end up calling `buildGraphQlDocument()` anyway.